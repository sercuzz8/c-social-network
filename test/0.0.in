addent "alice"
addent "bruno"
addent "carlo"
addent "dario"
report
addrel "carlo" "bruno" "amico_di"
report
addrel "carlo" "alice" "amico_di"
report
addrel "alice" "bruno" "amico_di"
report
addrel "bruno" "dario" "compagno_di"
report
delrel "carlo" "alice" "amico_di"
report
addrel "carlo" "alice" "compagno_di"
report
addrel "carlo" "bruno" "compagno_di"
report
delent "alice"
report
end
