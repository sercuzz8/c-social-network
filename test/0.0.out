none
"amico_di" "bruno" 1;
"amico_di" "alice" "bruno" 1;
"amico_di" "bruno" 2;
"amico_di" "bruno" 2; "compagno_di" "dario" 1;
"amico_di" "bruno" 2; "compagno_di" "dario" 1;
"amico_di" "bruno" 2; "compagno_di" "alice" "dario" 1;
"amico_di" "bruno" 2; "compagno_di" "alice" "bruno" "dario" 1;
"amico_di" "bruno" 1; "compagno_di" "bruno" "dario" 1;